README.md

```diff
@@ -9,6 +9,8 @@
 - Support most popular **Linux distributions**
 - **Continuous integration tests**
 
+To create a cluster in vagrant simply run `vagrant up`
+
 For an easy way to use it, check out [**kargo-cli**](https://github.com/kubespray/kargo-cli) </br>
```

Comment:
Vagrant is not limited to test. 
If you have access to some hypervisors on premise on want to use them 'like' in a cloud, vagrant is nice tool to spawn them.